RadixWallet/Features/ImportMnemonic/ImportMnemonic.swift

```diff
@@ -20,12 +20,10 @@ public struct ImportMnemonic: Sendable, FeatureReducer {
 			return wordCount
 		}
 
-		public mutating func changeWordCount(by delta: Int) {
-			let positiveDelta = abs(delta)
-			precondition(positiveDelta.isMultiple(of: ImportMnemonic.wordsPerRow))
-
+		public mutating func changeWordCount(to newWordCount: BIP39.WordCount) {
 			let wordCount = words.count
-			let newWordCount = BIP39.WordCount(wordCount: wordCount + delta)! // might in fact be subtraction
+			let delta = newWordCount.rawValue - wordCount
+			let positiveDelta = abs(delta)
```

Comment:
Since we're never in a situation where we need `positiveDelta` and without knowing the sign of delta, I'd rather just use `-delta` when we know it's negative.

Reply:
fixed in https://github.com/radixdlt/babylon-wallet-ios/pull/935